node-commons/src/assembly/resources/conf/iotdb-common.properties

```diff
@@ -366,6 +366,10 @@ cluster_name=defaultCluster
 # Datatype: boolean
 # meta_data_cache_enable=true
 
+# Whether to enable query metrics monitoring or not. Enable this may cause a little query performance degradation.
+# Datatype: boolean
+# query_metric_enable = true
```

Comment:
I remember that @OneSizeFitsQuorum has already added a new level metric level which means no metrics at all, i think you can use that instead.

Reply:
Yes, just use DO_NOTNING level

Reply:
Change the level of metric to `DO_NOTHING` will also take some costs in cpu bound situation(TSBS-benchAnts), the query cost of QueryMetricsManager is (4+1+3+2+2+1+3+2+3+3+2+1+2+2+1+1+15+2+2+3+1+3+15+9+12)/3884=2.4%.

![image](https://user-images.githubusercontent.com/6756545/234169397-8a7b93fc-13cf-4649-b5c5-9f72bdfa3705.png)